src/pQueue.js

```diff
@@ -0,0 +1,67 @@
+'use strict'
+
+export default class PriorityQueue {
+  constructor(initialValues = []) {
+    this.elements = initialValues
+  }
+
+  enqueue(value, priority) {
+    this.elements[this.elements.length] = [value, priority]
+  }
+
+  front() {
+    if (this.elements.length){
+      let highest = this.elements[0]
+      for(let pair of this.elements) {
+        if (pair[1] > highest[1]) {
+          highest = pair
+        }
+      }
+      return highest
+    }
+    return null
+  }
+
+  back() {
+    if (this.elements.length){
+      let lowest = this.elements[0]
+      for(let pair of this.elements) {
+        if (pair[1] < lowest[1]) {
+          lowest = pair
+        }
+      }
+      return lowest
+    }
+    return null
+  }
+
+  dequeue() {
+    if (this.elements.length){
+      let highest = this.elements[0]
+      for(let pair of this.elements) {
+        if (pair[1] > highest[1]) {
+          highest = pair
+        }
+      }
+      this.elements.splice(this.elements.indexOf(highest),1)
+      return highest
+    }
+    return null
+  }
+
```

Comment:
You're repeating this "Find the max" "Find the min" code a few times here.  It would be a good idea to break these out into helper functions so they don't have to be repeated, and it helps make the code more easily readable, too.